<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta content="width=device-width, initial-scale=1.0" name="viewport">
    <title>Notes on RNA</title>
    <link crossorigin="anonymous" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-"
    rel="stylesheet">
    <!--[if lte IE 8]>
    <link rel="stylesheet" href="/css/layouts/side-menu-old-ie.css">
    <![endif]-->
    <!--[if gt IE 8]><!-->
    <link href="/css/layouts/side-menu.css" rel="stylesheet">
    <!--<![endif]-->
</head>
<body>

    <div id="layout">
        <!-- Menu toggle -->
        <a class="menu-link" href="#menu" id="menuLink">
            <!-- Hamburger icon -->
            <span></span>
        </a>

        <div id="menu">
            <div class="pure-menu">
                <a class="pure-menu-heading" href="../">RNA</a>

                <ul class="pure-menu-list">
                    <li class="pure-menu-list"><a class="pure-menu-link" href="../groups/groups.html">Groups</a></li>
                    <li class="pure-menu-item"><a class="pure-menu-link" href="rnafacts.html">Facts & News</a></li>
                    <li class="pure-menu-item"><a class="pure-menu-link" href="../motifs/motifs.html">Motifs</a></li>
                    <li class="pure-menu-item"><a class="pure-menu-link" href="../folding/folding.html">Folding</a></li>
                    <li class="pure-menu-item"><a class="pure-menu-link" href="../questions/questions.html">Questions</a></li>
                </ul>
            </div>
        </div>

        <div id="main">
            <div class="header">
                <h1>Facts & News</h1>
            </div>

            <div class="content">

<hr>
<b>Jesse Stombaugh, Craig L. Zirbel,  Eric Westhof and Neocles
  Leontis, NAR, 2009, 37, 2294-2312 </b>
<br>
<br>
The questions remains as to the ability of FR3D to go past rnaview in
  identification of base-pairs with the Leontis-Westhof
  annotation. That is, is it the case now that no N.A. base-pairs
  appear in the output? Hopefully it is.
<br>
<br>


<hr>
<b>Jin-Der Wen, Laura Lancaster, Courtney Hodges, Ana-Carolina Zeri,
  Shige H. Yoshimura, Harry F. Noller, Carlos Bustamante, Ignacio
  Tinoco, Nature, 452, 598-603, 3 April 2008</b>
<br>
DOI: <a href="http://dx.doi.org/10.1038/nature06716">10.1038/nature06716</a></b>
<br>
<br>
Using single  molecule spectroscopy Tinoco and collaborators have
followed the  translation  in a single ribosome of and mRNA made up of
a 60 bp-hairpin, a GUAA and  GUUU tetraloop, and a 49 nucleotide 5' ss
region. They have also used a larger mRNA composed of 274 b-p.
<br>
<br>

<hr>
<b>Naoko Abe, Hiroshi Abe and Yoshihiro Ito. JACS, 15 November 2007.</b>
<br>
DOI: <a href="http://dx.doi.org/10.1021/ja0754453">10.1021/ja0754453</a>
<br>
<br>
Ito  el al.  have synthesized  dumbbell RNA's  just as  had  been done
before  for  DNA's  by  Breslauer  et  al.  (Biochemistry,  1989,  28,
268-273). The utility  of this dumbbell RNA's is  in RNA interference,
they are proven to be more  stable to enzymatic degradation and at the
same time offer  longer RNAi acitivity since the  shape does not offer
loose ends for enzyme  degradation. They have synthesized and compared
four different  sized dumbbells, that is, Db-15,  Db-19, Db-23, Db-27,
where  the number refers  to the  size of  the double  stranded A-type
region, the hairloops at the ends of the dumbells are of 9nt size.
<br>
<br>

<hr>
<b>David R. Corey and Bethany Janowski, Nat. Chem. Biol. 2007</b>
<br>
<br>
Taken from C&EN February 5, 2007
<br>
Short  pieces of  double-stranded RNA  may  be able  to activate  gene
expression. They target DNA instead of targeting mRNA as it happens in
RNAi.
<br>
<br>

<hr>
<b>Gottesman et al. Science, 26 January 2007</b>
<br>
<br>
Taken from C&EN January 22, 2007
<br>
Gottesman  et  al.  have  found   that  silent  genes  are  no  longer
silent. They believe that the  resulting proteins will differ in shape
due to  the different  dynamics of their  folding. The  "silent" genes
will  act as  delayers, and  so the  final configurational  state that
proteins with the same aminoacid sequence will fold to, when different
synonymous genes are coding for the same aminoacid, will be different.
<br>
<br>

<hr>
<b>J. Chem. Theory Comput. 2006, 2, 1444-1452</b>
<br>
<br>
The influence of stacking on the HB donating potential was shown to be dominated by the stacking geometry and not by
the nature of the stacking partner. The  present  findings  suggest  that  the pi-pi interaction  on
itself does  not  have  an  overall strengthening on H bonding in DNA.
<br>
<br>

<hr>
<b>Moore and Steitz, Tr. in Bioch. Sci. June 2005</b>
<br>
<br>
There is still no atomic-resolution structure for a 70S ribosome in any of its states.
Consequently, we have no idea why the peptydil-transferase activity of the 70S
ribosome is ~10^4 times that of its isolated 50S subunit.
<br>
<br>

<hr>
<b>Onoa and Tinoco, Curr. Op. Str. Bio. 2003</b>
<br>
<br>
The ultimate application of a better understanding of RNA folding is to predict
the folded, partially folded and unfolded states of any RNA in physiological
environment.
<br>
<br>

<hr>
<b>Chen and Dill, PNAS, 2002</b>
<br>
<br>
The principle that emerges, for which there is growing experimental support, is
that although protein folding tends to involve highly cooperative two-state
thermodynamic transitions, without detectable intermediates, the folding of RNA
secondary structures may involve rugged landscapes, often with more complex
intermediate states.
<br>
<br>

<hr>
<b>Tianbing Xia, David H. Mathews and Douglas H. Turner, RNA, Pergamon Press 2001,
Thermodynamics of RNA Secondary Structure Formation</b>
<br>
<br>
Sequences with identical composition but different permutations of base pairs may have
different free energy changes for duplex formation. For example, the duplexes
(5'GUUCGAAC3')2 and (5'UUGGCCAA3')2 both have four AU and four GC base pairs, but have free energy
changes of duplex formation at 37 C of -8.8 and -11.0 kcal mol-1, respectively. This difference of
2.2 kcal mol-1 translates into a 35-fold difference in equilibrium constants for duplex formation at
37 C. <b>Thus, stability depends on more than the number of hydrogen bonds formed. Presumably,
vertical stacking interactions between neighboring base pairs are the sequence-dependent variable.</b>
<br>
<br>


        </div>

    </div>
</div>


<script src="/js/ui.js"></script>

</body>
</html>
